kin-sdk/kin-sdk-lib/src/test/java/kin/sdk/TransactionSenderTest.java

```diff
@@ -290,7 +291,7 @@ public void sendTransaction_ThirdQuery_ConnectionException() throws Exception {
     @Test(timeout = 500)
     public void sendTransaction_changeTimeOut() throws Exception {
         String url = mockWebServer.url("").toString();
-        server = new Server(url, 100, TimeUnit.MILLISECONDS);
+        server = new Server(url, new OkHttpClient.Builder().build());
```

Comment:
The timeout is now replaced with the default one of 10sec, this actually should have failed the test, as the test should be timeout after 500ms, but from some reason, Junit failed to timeout the test now while it's in master branch.